src/app/api/ai/route.ts

```
@@ -0,0 +1,102 @@
+            });
+            break;
+        default:
+            return Response.json({ error: "AI model not supported" }, { status: 403 });
+    }
+
+    switch (validatedAIOperations.data) {
+        case "search":
+            const result = streamText({
+                model: aiProvider(validatedAI.data.model),
+                messages,
+            });
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)